src/components/vote_summary_table/VoteSummaryTable.js

```diff
@@ -0,0 +1,130 @@
+import SimpleTable from "../SimpleTable";
+import {h} from "react-hyperscript-helpers";
+import {Styles} from "../../libs/theme";
+import {isNil} from "lodash/fp";
+import {useCallback, useEffect, useState} from "react";
+import {formatDate, sortVisibleTable} from "../../libs/utils";
+
+const styles = {
+  baseStyle: {
+    fontFamily: 'Montserrat',
+    fontSize: '1.4rem',
+    fontWeight: 400,
+    color: '#333F52',
+    display: 'flex',
+    padding: '1rem 2%',
+    lineHeight: '1.6rem',
+    justifyContent: 'space-between',
+    alignItems: 'center'
+  },
+  columnStyle: Object.assign({}, Styles.TABLE.HEADER_ROW, {
+    fontFamily: 'Montserrat',
+    fontSize: '1.4rem',
+    color: '#333F52',
+    justifyContent: 'space-between',
+  }),
+  cellWidths: {
+    vote: '10%',
+    name: '15%',
+    date: '10%',
+    rationale: '50%',
+  },
+};
+
+const columnHeaderFormat = {
+  vote: {label: 'Vote', cellStyle: {width: styles.cellWidths.vote}, sortable: true},
+  name: {label: 'Name', cellStyle: {width: styles.cellWidths.name}, sortable: true},
+  date: {label: 'Date', cellStyle: {width: styles.cellWidths.date}, sortable: true},
+  rationale: {label: 'Rationale', cellStyle: {width: styles.cellWidths.rationale}, sortable: true},
+};
+
+const columnHeaderData = () => {
+  const { vote, name, date, rationale } = columnHeaderFormat;
+  return [vote, name, date, rationale];
+};
+
+const processVoteSummaryRowData = ({ dacVotes }) => {
+  if(!isNil(dacVotes)) {
+    return dacVotes.map((dacVote) => {
+      const { vote, displayName } = dacVote;
+      const { voteId, createDate, updateDate, rationale } = vote;
```

Comment:
I'm not sure where this model is defined. When I'm looking at the votes that come in from a DAR Collection, they look like this:
```
{
  "dacUserId": 3396, 
  "displayName": "Jonathan Lawson (Admin & DAC Member)", 
  "electionId": 1480, 
  "type": "DAC", 
  "voteId": 20097
}
```
The `vote` value, which is a true|false, can also be null as in the case above, and might not be in the object. I tested this locally by passing in all of the votes for a specific election on a specific dar collection into this component.

Reply:
What page do you recommend adding the table to for testing locally as I make changes? I was using DarApplication since it displays dar election information , but that seems to use a different vote structure (found within the ElectionReview model) compared to the Vote model

Reply:
Great question - I used the dar collection page. I just used some test code to grab the votes on the first election on the first  dar that I was looking at and fed that into this component.

Reply:
You can see an example here where we dig down into the collection's elections to find votes the user for a user: https://github.com/DataBiosphere/duos-ui/blob/develop/src/libs/utils.js#L358